recipes/eups/deactivate.sh

```diff
@@ -16,10 +25,34 @@ WORK=${WORK//$REMOVE/:}
 WORK=${WORK%:}
 WORK=${WORK#:}
 export PATH=$WORK
+unset WORK
+unset REMOVE
+
 
-# remove EUPS vars
+# restore EUPS env variables existing prior to the activation
 for var in EUPS_PATH EUPS_SHELL SETUP_EUPS EUPS_DIR EUPS_PKGROOT; do
-    unset $var
+  unset $var
+  bkvar="BACKUP_$var"
+  if [[ "${!bkvar}" ]]; then
+    export $var="${!bkvar}"
+    unset "$bkvar"
+  fi
 done
+unset bkvar
+unset var
 unset -f setup
+if [[ "$BACKUP_setup" ]]; then
+  eval "$BACKUP_setup"
+  unset BACKUP_setup
+fi
 unset -f unsetup
+if [[ "$BACKUP_unsetup" ]]; then
+  eval "$BACKUP_unsetup"
+  unset BACKUP_unsetup
+fi
+
+
+# restoring exisisting python path
+export PYTHONPATH=${BACKUP_PYTHONPATH}
+unset BACKUP_PYTHONPATH
```

Comment:
I’d use a more specific prefix than backup. Something unique

Reply:
Jost for PYTHONPATH or for all the backup variables?

Reply:
All of them. Maybe CONDA_EUPS_BACKUP or somethjng like that